lib/instance-validator.js

```diff
@@ -321,10 +321,7 @@ class InstanceValidator {
     let error;
 
     if (rawAttribute.allowNull === false && (value === null || value === undefined)) {
-      const validators = this.modelInstance.validators[field];
-      const errMsg = validators
-                      ? (validators.notNull || {}).msg
-                      : `${field} cannot be null`;
+      const errMsg = `${field} cannot be null`;
```

Comment:
oh, that makes sense. In that case the way you did it is better. I also cannot think of a way where `${field} cannot be null` doesn't describe the exact problem.